mlflow/server/js/src/experiment-tracking/components/experiment-page/components/runs/ExperimentViewRunsControlsFilters.tsx

```diff
@@ -187,6 +188,25 @@ export const ExperimentViewRunsControlsFilters = React.memo(
               <Button type='tertiary' icon={<OverflowIcon />} />
             </Dropdown>
             <ExperimentViewRefreshButton />
+            {/* TODO: remove this */}
+            {/* TODO: create a basic button to make an api request */}
+            <Button
+              onClick={() => {
+                MlflowService.proxyQueryGatewayRoutes(
+                  {
+                    host: "127.0.0.1",
+                    port: "5001",
+                    request_type: "GET",
+                    gateway_path: "api/2.0/gateway/routes"
```

Comment:
We unfortunately won't be able to hardcode this in the frontend since the gateway host / port is user-configurable